src/components/v5/common/ActionSidebar/partials/forms/ManageTokensForm/utils.tsx

```diff
@@ -24,3 +28,35 @@ export const manageTokensDescriptionMetadataGetter: DescriptionMetadataGetter<
     isMotion: true,
   });
 };
+
+export const getManageTokensPayload = (
+  colony: Colony,
+  values: ManageTokensFormValues,
+) => {
+  const {
+    selectedTokenAddresses,
+    description: annotationMessage,
+    title,
+  } = values;
+
+  let addresses = selectedTokenAddresses.map(({ token }) => token) ?? [];
```

Comment:
maybe we can do just 
```JSX
const tokenAddresses =  [
  ...new Set(
    selectedTokenAddresses.map(({ token }) => token)
      .map((address) => createAddress(address))
      .filter((address) => {
        return (
          address !== ADDRESS_ZERO &&
          address !== colony.nativeToken.tokenAddress
        );
      }),
    ),
  ];
```
we can always fallback to empty array if selectedTokenAddresses can be undefined a la 
`(selectedTokenAddressed || []).map` but as I see it's always defined

Reply:
```JSX
const tokenAddresses =  [
  ...new Set(
    selectedTokenAddresses
      .map(({ token }) => createAddress(token))
      .filter((address) => {
        return (
          address !== ADDRESS_ZERO &&
          address !== colony.nativeToken.tokenAddress
        );
      }),
    ),
  ];
```
would be even a bit more concise :)